.tekton/pipeline/rhtap-staging-e2e.yaml

```diff
@@ -0,0 +1,156 @@
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: rhtap-staging-e2e
+spec:
+  params:
+    - name: repo_url
+    - name: revision
+    - name: SEND_SLACK_NOTIFICATION
+      description: sends failed or success message to slack if the param is set to True/true
+      default: "false"
+  workspaces:
+    - name: git
+    - name: shared-workspace
+  tasks:
+    - name: staging-argocd-login
```

Comment:
per exchange in slack, I think this needs to send a slack message to appropriate channels when this flow starts

Reply:
As this change is specific for staging so IMO syncing one by one should not affect team. Also staging is meant to be dynamic, considering this I would rather avoid notifying people.

Reply:
I'm not sure staging being "dynamic" should preclude announcements like this, as there is "official" activities outside of users like performance testing that could be affected by this.

Sorry @prietyc123 I'm not sure I'm ready to dismiss this.  I'd like to discuss with a broader audience.

I've unresolved this for the time being.